index-hic.html

```diff
@@ -126,21 +122,26 @@ <h3 id="contact-matrix-as-a-basis">Contact matrix as a basis</h3>
   <li><a href="https://hicexplorer.usegalaxy.eu/u/joachim-wolff/w/plot-tads-and-pc">Plot TADs and PC</a></li>
 </ul>
 
-
-
-  <iframe style="border: 0px" width="100%" height="150px" src="https://stats.usegalaxy.eu/dashboard-solo/db/jobs-dashboard?panelId=1&orgId=1" ></iframe>
-
-
-  <div class="row">
-
+<h3 id="data-policy">Data policy</h3>
+<p>
+For unregistered users all uploaded and processed data will only be accessible during one browser session,
+that is if the correct browser cookie is accessible. If the datasets are not accessed for 90 days it will be
+deleted. For registered users the data is available as long as they are not actively delete by the user.
+If it is deleted by the user or a user deletes its account we delete the data within 7 days permanently.
+An extended quota can be
+<a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ" target="_blank">requested</a> for a limited time period in special cases.
+We are located in the European Union, therefore the Galaxy service will be compliant to the EU General Data Protection
```

Comment:
Just FYI, this is not the reason we will be compliant. We must be compliant because *we process the data of EU citizens*, it applies to US companies as well.